CMakeLists.txt

```diff
@@ -108,19 +120,16 @@ endif ()
 # Targets
 ################################################################################
 
-if (LIBSBP_SEARCH_PATH)
-  include_directories("${LIBSBP_SEARCH_PATH}/include")
-else (LIBSBP_SEARCH_PATH)
-  if(EXISTS ${CMAKE_SOURCE_DIR}/libsbp/c)
-    include_directories("${PROJECT_SOURCE_DIR}/libsbp/c/include")
-  endif()
-endif(LIBSBP_SEARCH_PATH)
-
-if(EXISTS ${PROJECT_SOURCE_DIR}/libswiftnav)
-  include_directories("${PROJECT_SOURCE_DIR}/libswiftnav/include")
-  add_subdirectory("${PROJECT_SOURCE_DIR}/libswiftnav")
-endif()
+option(libsbp_ENABLE_TESTS "" OFF)
```

Comment:
Is the intention here to always disable the libsbp tests, or just disable them by default? When I see an `option` in CMake I immediately think this is something I should consider setting/changing. When I see a `set` I'm less inclined to consider modifying the value when I invoke `cmake`.

Reply:
The intent is that when a project includes something else as a dependency it disabled unit tests in that dependency. Normally test suites are enabled by default, this line will disable the libsbp unit tests by default when libsettings includes it in the build tree.

It is possible to override this on the command line

cmake -Dlibsbp_ENABLE_TESTS=ON ..